.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Avocado Gating CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Logical, stylistic, analytical and formatter checks
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
```

Comment:
Do you anticipate if those action versions would be deprecated/removed in the future?  Does GH say anything about that?

Reply:
I have to investigate more, but afaict, this is a "verified partner": 

> Actions with the badge indicate GitHub has verified the creator of the action as a partner organization. 

Also, this one has more than 1.5k stars and IIUC it is maintained by Github. I guess this is just a matter of updating the action dependency or replacing in case of deprecation.